src/pages/ProductDetails/ProductDetails.js

```diff
@@ -34,83 +36,98 @@ const ProductDetails = props => {
     fetchProductDetails();
   }, [productID]);
 
+  const getSimilarProducts = () => {
+    return fuseSearch(['category'])
+      .search(category)
+      .map(({ item }) => item);
```

Comment:
Is this map really required?

Reply:
Yeah because the fuse returns a piped array which is not compatible with our Component props.
Something like this
`
[
   { item: { } },
   { item: { } },
]
`

Reply:
Ah, I see, makes sense. Good to go here, then.